Sources/SwiftDriver/Driver/Driver.swift

```diff
@@ -2005,8 +2005,11 @@ extension Driver {
 
       if !fileSystem.exists(path) {
         diagnosticsEngine.emit(.warning_no_such_sdk(sdkPath))
+      } else if targetTriple != nil,
+                isSDKTooOld(sdkPath: path, target: targetTriple!) {
```

Comment:
```suggestion
      } else if let targetTriple = targetTriple,
                isSDKTooOld(sdkPath: path, target: targetTriple) {
```
Don't have to force unwrap here if you use `if let`

Reply:
This code looks a bit different now, but thank you for taking a look.